experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -274,7 +274,10 @@ describe("Editing", () => {
 			expectJsonTree(tree1, ["B", "A"]);
 		});
 
-		it("can rebase intra-field move over insert", () => {
+		// This test fails due to the rebaser incorrectly ordering the cell created by the insert of C before the cell
```

Comment:
Why did your change affect this test? It looks like there's only one move of a single node, so I don't see how changing the delta move ID could affect the outcome.

Reply:
The move ID collision bug and bug 5351 combined in a way that made the test pass. That's right, two wrongs do make a right sometimes.
More specifically, the changeset produce to update the local state include a ReturnFrom with ID 0 and revision foo (but no ReturnTo), and a MoveIn with ID 0 and revision bar (but no MoveOut). The toDelta used to (mistakenly) treat these two as part of the same move.